### PR TITLE
Add unity column, ability to filter unity aligned studies and filter by antibody target

### PR DESCRIPTION
**Pull request by atatmaja**
## Briefly describe the feature or bug that this PR addresses.

- Adds is_unity_aligned column to DB and allows for filtering based on that.
- Adds filtering by antibody target
- Moves get_all_filter_options function to data provider service, as it is not related to the etl

## Please link the Airtable ticket associated with this PR.

https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/recUvxiYC1O0AZE5p?blocks=bipIkdtYHmboUfJut
https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/recGnymO05NXQZDqX?blocks=bipIkdtYHmboUfJut

## Describe the steps you took to test the feature/bugfix introduced by this PR.

- Used postman to test.
- Note if we end up merging the testing PR before this I will add some tests that address this PR

## Does any infrastructure work need to be done before this PR can be pushed to production?

No

### CODE REVIEW

File: app/database_etl/airtable_records_handler/airtable_records_formatter.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def standardize_airtable_data(df: pd.DataFrame) -> pd.DataFrame:
                                 'ind_se', 'ind_se_n', 'ind_sp', 'ind_sp_n', 'jbi_1', 'jbi_2', 'jbi_3', 'jbi_4',
                                 'jbi_5', 'jbi_6', 'jbi_7', 'jbi_8', 'jbi_9', 'measure_of_age', 'number_of_females',
                                 'number_of_males', 'test_linked_uid', 'average_age',
-                                'test_not_linked_reason', 'include_in_srma']
+                                'test_not_linked_reason', 'include_in_srma', 'is_unity_aligned']
```
Comment (atatmaja) [inline app/database_etl/airtable_records_handler/airtable_records_formatter.py:65]: Note: Christian created a lookup field for me in the estimates table for the UNITY: Criteria field. That's why it shows up as a single element list col.

**Comment by rahularoradfs** [comment]
it bugs me that we've got to change 11 files in order to add / modify these fields!! schema improvements would be very high value

File: app/namespaces/data_provider/data_provider_controller.py [hunk 2]
```diff
@@ -58,7 +58,8 @@ def post(self):
                                       prioritize_estimates_mode=prioritize_estimates_mode,
                                       include_in_srma=include_in_srma,
                                       include_disputed_regions=include_disputed_regions,
-                                      include_subgeography_estimates=include_subgeography_estimates)
+                                      include_subgeography_estimates=include_subgeography_estimates,
+                                      unity_aligned=unity_aligned)
```
Comment (rahularoradfs) [inline app/namespaces/data_provider/data_provider_controller.py:62]: maybe we could rename this parameter to be slightly more clear? include_only_unity_aligned or unity_aligned_only

**Review by rahularoradfs — APPROVED** [review]
looks good!! one naming note

File: app/namespaces/data_provider/data_provider_controller.py (hunk 2 above)
Comment (atatmaja) [reply app/namespaces/data_provider/data_provider_controller.py:62]: let's do `unity_aligned_only` don't usually love long parameter names but this is justified

**Pull request merged**